content/en/docs/reference/glossary/cloud-provider.md

```diff
@@ -1,18 +1,32 @@
 ---
-title: Cloud Provider
+title: Cloud provider
```

Comment:
Hi @sftim . A few more bits of feedback 😃 .

- The title should use the same case as other titles in the glossary. If you want to change the way the
titles are capitalized in the glossary, then propose another way. Same idea for when to add a period at the end of a short description sentence. These are excessive nits, but I think it helps the glossary look consistent and makes it easier for authors to follow a pattern. The `aka` or metadata field is not heavily used in the glossary. I think the format of the aka field (shortcode?) looks odd wrt the rest of the content (personal opinion), but other wise this is OK.
- Line 24: I _think_ the k8s docs conform to American English. So *while* I like whilst, while is probably more acceptable.

Reply:
Ah, I don't speak American so well and I didn't know that whilst looks odd!